ci/utils.py

```diff
@@ -178,7 +178,7 @@ def get_existing_bundle_list(model_info):
 
 
 def create_bundle_to_huggingface(bundle_name: str, org_name: str):
```

Comment:
Not a blocker, but I personally think such tool can be included in the monai.bundle, so that the user can publish their bundle to HF easily.